docs/managing-data-quality/monitoring-failed-events/ui/index.md

```diff
@@ -1,14 +1,15 @@
 ---
 title: "Monitoring failed events in the Console"
 date: "2020-06-03"
+sidebar_label: "Using the UI"
 sidebar_position: 2000
 ---
 
 Note
 
 This functionality is only available for pipeline versions R118+. If you are unsure of which version your pipeline is running, please contact support.
```

Comment:
Why isn't this the reusable block, and again weird single note word.

Reply:
I need to remove this, thanks